### DISCUSSION TITLE
Looking for help in quiz

### DISCUSSION THREAD

**Discussion by Recordarses** [microsoft/Data-Science-For-Beginners#551] [Q&A]
# Hello, my friends.
I can't find the location of quiz, I also searched the quiz-app, but nothing helped.
Could you show me the link, please?
Very thanks!

**Comment by Zetrio16** [comment]
Isn't that in src/components inside quiz-app? Named "quiz.vue".

**Reply by Recordarses** [reply]
Thanks for your reply! 
I'm new to computers and just getting started, and there are a lot of programs I don't quite understand. I have just solved the problem by using VsCode's terminal and typing in the appropriate command to get the link.
Thanks again for your reply.

**Reply by Zetrio16** [reply]
It will gradually become easier and understandable. Don't worry. 😉
